test/integration/transactions_test.go

```diff
@@ -99,7 +99,6 @@ func initListeners(t *testing.T, account *sdk.Account, hash *sdk.Hash, tx sdk.Tr
 		fmt.Println("Successful!")
 		transaction.GetAbstractTransaction().Signer = nil
 		transaction.GetAbstractTransaction().Signature = ""
-		transaction.GetAbstractTransaction().TransactionInfo = nil
```

Comment:
It will be fixed in PR: https://github.com/proximax-storage/go-xpx-chain-sdk/pull/68